### PR TITLE
Implement New toolkit version csharp samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
**Sample List  -** 
        **app-anonymous-users/csharp
         bot-sequential-flow-adaptive-cards/csharp
        meeting-recruitment-app/csharp**

### CODE REVIEW

**Review by Alive-Fish — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json [hunk 1]
```diff
@@ -19,13 +19,13 @@
     "full": "Anonymous User Support"
   },
   "description": {
-    "short": "Enable anonymous user participation in Teams meeting apps with this sample.",
-    "full": "This sample demonstrates how to support anonymous users in Microsoft Teams meeting apps, showcasing the setup and interaction flow for integrating guest and tenant users seamlessly."
+    "short": "This sample shows anonymous users support in Teams meeting apps.",
+    "full": "This sample shows anonymous users support in Teams meeting apps."
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json]: Updated Below Comments - 
**This sample app enables anonymous user support in Microsoft Teams meeting apps using Azure AD, Bot Framework, and tunneling setup, allowing guest users to interact seamlessly through features like meeting tabs and stage views.**

File: samples/app-anonymous-users/csharp/Readme.md [hunk 2]
```diff
@@ -31,6 +31,27 @@ This sample demonstrates how to enable anonymous user support in Microsoft Teams
...
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio.
+
+1.Install Visual Studio 2022 Version 17.10 Preview 4 or higher Visual Studio
+2.Install Teams Toolkit for Visual Studio Teams Toolkit extension
+3.In the debug dropdown menu of Visual Studio, select Dev Tunnels > Create A Tunnel (set authentication type to Public) or select an existing public dev tunnel.
+4.In the debug dropdown menu of Visual Studio, select default startup project > Microsoft Teams (browser)
+5.In Visual Studio, right-click your TeamsApp project and Select Teams Toolkit > Prepare Teams App Dependencies
+6.Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps.
+7.To test facebook auth flow create a facebookapp and get client id and secret for facebook app. Now go to your bot channel registartion -> configuration -> Add OAuth connection string
```
Comment (Pawank-MSFT) [inline samples/app-anonymous-users/csharp/Readme.md:46]: "registartion" should be corrected to "registration".

File: samples/app-anonymous-users/csharp/Readme.md [hunk 3]
```diff
@@ -31,6 +31,27 @@ This sample demonstrates how to enable anonymous user support in Microsoft Teams
...
+
+Provide connection Name : for eg facebookconnection. You'll use this name in your bot in the appsettings.json file.
+Select service provider ad facebook
```
Comment (Pawank-MSFT) [inline samples/app-anonymous-users/csharp/Readme.md:49]: "Select service provider ad facebook" should be corrected to "Select service provider as Facebook."

File: samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
   "manifestVersion": "1.16",
```
Comment (Pawank-MSFT) [inline samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json]: please refer latest manifest version.

File: samples/meeting-recruitment-app/csharp/TeamsApp/appPackage/manifest.json [hunk 5]
```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.16",
```
Comment (Pawank-MSFT) [inline samples/meeting-recruitment-app/csharp/TeamsApp/appPackage/manifest.json:3]: Please update latest manifest version

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix given comments

File: samples/app-anonymous-users/csharp/Readme.md (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/csharp/Readme.md:46]: Updated - "registration"

File: samples/app-anonymous-users/csharp/Readme.md (hunk 3 above)
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/csharp/Readme.md:49]: Updated - "Select service provider as Facebook"

File: samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json (hunk 4 above)
Comment (Harikrishnan-MSFT) [reply samples/app-anonymous-users/csharp/TeamsApp/appPackage/manifest.json]: Updated already - 
image

File: samples/meeting-recruitment-app/csharp/TeamsApp/appPackage/manifest.json (hunk 5 above)
Comment (Harikrishnan-MSFT) [reply samples/meeting-recruitment-app/csharp/TeamsApp/appPackage/manifest.json:3]: Updated - 19 version

**Review by Jegadeesh-MSFT — APPROVED** [review]
Looks Correct !

**Pull request merged**